recipes/openms-meta/build.sh

```
@@ -23,15 +209,17 @@ fi
 
 # not sure if needed. CMake should take care of that.
 LDFLAGS='-Wl,-rpath,${RPATH}'
+# Note: Cmake could not find Qt6Config.cmake on fedora without adding those prefix paths
 
 cmake .. \
+  -G Ninja 
   -DOPENMS_GIT_SHORT_REFSPEC="release/${PKG_VERSION}" \
```

Shellcheck (0.10.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)